recipes/libffi/all/conanfile.py

```diff
@@ -37,7 +37,7 @@ def config_options(self):
 
     def build_requirements(self):
         if tools.os_info.is_windows and "CONAN_BASH_PATH" not in os.environ:
```

Comment:
We prefer using the tools helpers

```suggestion
        if tools.os_info.is_windows and not tools.get_env("CONAN_BASH_PATH"):
```